packages/kbn-utility-types/src/dot_test.ts

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { expectNotType, expectType } from 'tsd';
+import { DedotObject, DotObject } from './dot';
```

Comment:
Can't this file be moved under `./src/tsd_tests/test_d` where the other type tests are?